blockchain/source/contracts/ProfileRegistry.sol

```diff
@@ -8,6 +8,14 @@ contract ProfileRegistry {
         _;
     }
 
+    uint constant ANONIMOUS = 1;
```

Comment:
it's quite inconsistent with 
https://github.com/sonm-io/market/blob/master/contracts/Market.sol#L34